project_euler/problem_03/sol04.py

```diff
@@ -31,21 +31,12 @@ def solution(n):
     Traceback (most recent call last):
         ...
     ValueError: Parameter n must be greater or equal to one.
-    >>> solution([])
-    Traceback (most recent call last):
-        ...
-    TypeError: Parameter n must be int or passive of cast to int.
-    >>> solution("asd")
-    Traceback (most recent call last):
-        ...
-    TypeError: Parameter n must be int or passive of cast to int.
     """
-    try:
-        n = int(n)
-    except (TypeError, ValueError):
-        raise TypeError("Parameter n must be int or passive of cast to int.")
```

Comment:
Why remove these check?  We can specify types but callers are free to ignore them.

Reply:
Exactly, my change request was to remove `TypeError` **only**